test/scenarios/sflight/integration.test.js

```diff
@@ -2,6 +2,11 @@ const childProcess = require('child_process')
 const path = require('path')
 const cds = require('../../cds')
 
+// REVISIT: @capire/sflight tests don't expect IEEE754 compliant responses, but do send the IEEE754 header according to the test
+// As it is not possible to configure the IEEE754 flag for @cap-js/hana it is required to inject this configurable:true
+// As the cqn2sql output converter injection doesn't allow for it to be overwritten at a later stage, but configurable will stay true
+Object.defineProperty(cds.builtin.types.Decimal.constructor.prototype, "CQN2HANA:convertOutput", { get: () => a => a, configurable: true })
```

Comment:
What is this test suite doing if no tests are run?

Reply:
This test suite is importing the `jest` tests from the `sflight` repository ([here](https://github.com/cap-js/cds-dbs/pull/673/files/7cfa9374865d51ecd2224d3b90983e3c3889b622#diff-345b318913531034f8b42e7b87680c450bc1c9a00b255f4ddd804f4299b505d9R24)). Which contain the wrong expectations as they claim to request `IEEE` compliance, but don't expect them and don't receive them.

Reply:
isn't the sflight tests to be adapted instead?

Reply:
yes that is why the `REVISIT` comment is there

Reply:
Can you please also craete a follow up?
I don't find the time to prepare the tests.